tutorials/s4sdk-secure-cloudfoundry/s4sdk-secure-cloudfoundry.md

```diff
@@ -90,168 +69,148 @@ This alternative approach requires that you have npm installed on your machine.
 cf unbind-service firstapp my-xsuaa
 cf delete-service my-xsuaa​
 ```
 
-`7.` Before we deploy your folder structure should look similar to this:
+Then deploy the app router using the following (with the appropriate API endpoint of your Cloud Foundry region):
 
-![folder](Figure4-1.png)
-
-You may have different content in the **approuter** folder depending on the download alternative you have chosen. Ensure that **`xs-app`.json** and **`package`.json** and the **node`_modules`** directory with content are present.
-
-`8.` Then deploy the AppRouter using the following (with the appropriate API endpoint of your Cloud Foundry region):
-```
+```bash
 cd <destLocation>
-cf api https://api.cf.eu10.hana.ondemand.com
-cf login
 cf push
 ```
-`9.` Afterwards you should be able to locate the App Router from within your browser using the host name of your deployment. In my case this is `https://approuter-p1942765239trial.cfapps.eu10.hana.ondemand.com/hello` which should face you with the following login page where you can use your user e-mail and password:
+
+Afterwards you should be able to locate the app router from within your browser using the host name of your deployment. In my case this is `https://approuter-p1942765239trial.cfapps.eu10.hana.ondemand.com/hello` which should face you with the following login page where you can use your user e-mail and password:
 
 ![login](Figure5-1.png)
 
-`10.` After logging in you should see the `HelloWorld` servlet which is now served by the App Router as a proxy to your Java application:
+After logging in you should see the `HelloWorld` servlet which is now served by the App Router as a proxy to your Java application:
 
 ![helloworld](Figure6-1.png)
 
 [DONE]
 [ACCORDION-END]
 
-[ACCORDION-BEGIN [Step 5: ](Protect your backend microservice)]
-After authentication works with the App Router, your java backend service is still fully visible in the web and not protected. We, therefore, need to protect our java microservices as well so that they accept requests with valid `JWTs` for the current user only. In addition, we will setup the microservice in a way that it deals with authorization, i.e., understands the OAuth scopes from the JWT that we have configured previously using the **`xs-security`.json** file.
-
-In the following, we will use the [Spring Security framework](https://spring.io/projects/spring-security) to protect the microservices. You can also use standard mechanisms of the SAP Java `buildpack` to achieve the same. If you do not want to use Spring Security please follow the steps [here](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/ead7ee64f96f4c42bacbf0ae23d4135b.html), nonetheless, the concepts described hereinafter apply for both methods.
-
-[DONE]
-[ACCORDION-END]
-
-[ACCORDION-BEGIN [Step 6: ](Install XS Security libs to your local Maven repository:)]
-The first step is to get some additional Java libs from Service Marketplace. To get them, do the following steps:
```

Comment:
This is still required, isn't it, if you use the dependencies listed in the pom below. For example, `com.sap.xs2.security` is not on Maven Central.

However, the setup should be easier with the approach described here: [with SAP buildpack](https://github.com/SAP/cloud-security-xsuaa-integration/tree/master/samples/sap-java-buildpack-api-usage) or here: [with Spring security](https://github.com/SAP/cloud-security-xsuaa-integration/tree/master/spring-xsuaa)